docs/docs-content/vm-management/create-manage-vm/advanced-topics/migrate-vm-kubevirt.md

```diff
@@ -80,6 +80,64 @@ from VMware vSphere to Palette VMO.
   - The Palette CLI must have access to both the VMO cluster and the machines to be migrated.
 - The kubectl command-line tool should also be installed. Refer to the
   [kubectl installation](https://kubernetes.io/docs/tasks/tools/install-kubectl/) guide to learn more.
+- We recommend providing a VMware Virtual Disk Development Kit (VDDK) image for the migration. This will significantly
+  speed up the migration.
+
+  - The VDDK image must be built and uploaded to your image registry before starting the migration. Refer to the
+    [Red Hat Documentation](https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/2.6/html/installing_and_using_the_migration_toolkit_for_virtualization/prerequisites_mtv#creating-vddk-image_mtv)
+    for guidance.
+  - The host with the Palette CLI setup for migration must have access to your image registry.
+  - If you are using a private image registry, you must create a Secret to be used for the migration. The Secret must be
+    in the form of a YAML file and the `metadata.name` value must be `vddk-image-pull-secret`.
+
+    <!--prettier-ignore-->
+    <details>
+    <summary> Example Secret Creation </summary>
+
+    A Secret can be created by issuing the following command.
+
+    ```shell
+    kubectl create secret docker-registry vddk-image-pull-secret \
+    --docker-server=myRegistryServer \
+    --docker-username=myUsername \
+    --docker-password=myPassword \
+    --docker-email=myEmail \
+    -o yaml > image-pull-secret.yaml
+    ```
+
+    This creates the `image-pull-secret.yaml` file in your working directory.
+
+    ```yaml title="Example contents of image-pull-secret.yaml"
+    apiVersion: v1
+    kind: Secret
+    metadata:
+      name: vddk-image-pull-secret
+    data:
+      .dockerconfigjson: #base64 encoded dockerconfigjson
+    type: kubernetes.io/dockerconfigjson
+    ```
+
+    The `data..dockerconfigjson` value contains your registry credentials, which have been base64 encoded by the
+    command.
+
+    Alternatively, you can manually encode a `config.json` by issuing the following command.
+
+    ```shell
+    cat path/to/config.json | base64 --wrap=0
+    ```
+
+    ```text title="Example output"
+    eyJodHRwczovL2luZGV4L ... J0QUl6RTIifX0=
+    ```
+
+    You can then use this output to create your own Secret manually. Just ensure that the `metadata.name` is set to
```

Comment:
```suggestion
    You can then use this output to create your own Secret manually. Ensure that the `metadata.name` is set to
```
Just sounds a bit informal, we can be more explicit I think and start with Ensure 😄